src/components/animated/_click.less

```diff
@@ -1,8 +1,13 @@
 .click-btn {
   box-shadow: 0px 5px 0px @lightGray;
 
+  &:focus {
+    box-shadow: 0px 5px 0px @lightGray;
+  }
+
   &:active {
```

Comment:
You still haven't made the change I requested regarding `&:active` and `&:focus` please go check it again and add it.

Reply:
actually this time i have made changes ...to work those buttons in intended manner...without writing any extra code